format-adapter.lua

```diff
@@ -16,7 +16,9 @@ local function syntax_convert(tree)
   elseif tree.kind == "literal" then
     return metalanguage.value(tree.anchor, {type = tree.literaltype, val = tree.val})
   elseif tree.kind == "string" then
-    error "syntax contains a string which isn't yet implemented"
+		-- FIXME: is there an issue with syntax or should elements[*].val get concated?
```

Comment:
@bootstrap-prime was it intended that format creates a list here? If no, can fix?

Reply:
Discussed and and reworked